src/clr-core/internal/base/focus-trap.base.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { html, LitElement } from 'lit-element';
+import { property } from '../decorators/property';
+import { FocusTrapTracker } from '../services/focus-trap-tracker.service';
+import { addAttributeValue, isHTMLElement, removeAttributeValue } from '../utils/dom';
+import { isBrowser } from '../utils/exists';
+
+export class CdsBaseFocusTrap extends LitElement {
+  private previousFocus: Element;
+  private topReboundEl: any;
+  private bottomReboundEl: any;
```

Comment:
The `topReboundEl` and `bottomReboundEl` can be so I'll update those. Not the previousFocus though because `document.activeElement`'s return type is `Element`.